deploy/operator/crd/healthcheck-crd.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: healthchecks.aspnetcore.ui
+
+spec:
+  group: aspnetcore.ui
+  names:
+    plural: healthchecks
+    singular: healthcheck
+    kind: HealthCheck
+    listKind: HealthChecks
+    shortNames:
+      - hc
+  versions:
+    - name: v1
+      served: true
+      storage: true
+
+  scope: Namespaced
+
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            name:
+              type: string
+            portType:
+              type: string
+              pattern: ^(LoadBalancer|ClusterIP|NodePort)$
+            portNumber:
+              type: number
+            uiPath:
+              type: string
+            servicesLabel:
+              type: string
+            servicesHealthPathLabel:
```

Comment:
I had made these annotation only (and also added an annotation for the port) in the discovery service.

Reply:
As we merged yesterday your changes, I should adapt this to match the discovery service. I'll turn this into annotations